src/net/highwayfrogs/editor/file/standard/psx/PSXMatrix.java

```diff
@@ -121,9 +121,6 @@ public double getRollAngle() {
      * @param roll  The new roll.
      */
     public void updateMatrix(double yaw, double pitch, double roll) {
-        if (Math.abs(pitch) >= Math.PI / 2)
```

Comment:
I should clarify. Set the pitch to either -90 or +90. Then, modify a slider for Yaw or Roll, and the model will spazz out.

Reply:
The correct solution for this may actually just be to limit the value on that specific function call...
The reason is that I believe the math is correct there, and the solution is legitimately to just prevent it from reaching this value.

Reply:
This is from Gimbal lock iirc.

Reply:
That makes sense. You are correct, it will help prevent Gimbal lock when working with Euler angles and rotations. I do agree that this check needs to happen, but at a more appropriate location. Leave this PR Open for now and I'll take a look in the next day or two.